source/group_eq_bot/configurations/sample.yaml

```diff
@@ -6,11 +6,11 @@ validation:
   limit_of_attempts_to_fail_per_question: 3
   mathematical_answer: 40
   questions_paths:
-    first: ${hydra:runtime.cwd}/assets/first_question.m4a
-    second: ${hydra:runtime.cwd}/assets/second_question.m4a
-    third: ${hydra:runtime.cwd}/assets/third_question.m4a
-    fourth: ${hydra:runtime.cwd}/assets/fourth_question.m4a
-    goodbye: ${hydra:runtime.cwd}/assets/say_goodbye.m4a
+    first: ${hydra:runtime.cwd}/assets/text/first_question.txt
+    second: ${hydra:runtime.cwd}/assets/audio/second_question.m4a
+    third: ${hydra:runtime.cwd}/assets/audio/third_question.m4a
+    fourth: ${hydra:runtime.cwd}/assets/text/fourth_question.txt
+    goodbye: ${hydra:runtime.cwd}/assets/text/say_goodbye.txt
```

Comment:
We've had a discussion about stripping the `audio` and `text` question in  configurations as:
```
- conversation
    - questions
        - audio
            - ....
        - video
        - text
```

Reply:
Applied